src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider making 'colorSupplier' a private field and initializing it at the class level to follow the best practice of defining class-level variables with appropriate access modifiers.